packages/formation/js/sidenav.js

```diff
@@ -97,15 +89,10 @@ class SideBarMenu {
     this.loadKeyNodes(trigger);
     this.menu.classList.add('va-sidebarnav--opened');
     this.menu.setAttribute('aria-hidden', 'false');
-    document
```

Comment:
We are no longer hiding the trigger button when the menu is open. This may have been done because of a layering issue; the trigger appeared in front of the menu itself. But who knows.